.gitignore

```diff
@@ -110,3 +110,4 @@ flopy/mf6/data/toml/
 
 # uv lockfile
 uv.lock
+/.vs
```

Comment:
can you put this up near `.vscode` under IDE files? also don't need the leading slash